_source/_posts/2024-01-15-spring-boot-graphql-react.md

```diff
@@ -1284,6 +1268,8 @@ Get an access token using the Auth0 CLI with the `auth0 test token` command:
 auth0 test token -a https://<your-auth0-domain>/api/v2/
 ```
 
+Select the **CLI Login Testing** app or any available client when prompted, you don't need to select any scope. You will also be prompted to open a browser window and log in with a user credential.
```

Comment:
I think it's better to add `-s openid` as a scope so the user isn't confused when they're prompted for one.